server/controllers/routes/admin/project/addProject.js

```diff
@@ -0,0 +1,32 @@
+const { addProject } = require('../../../../database/queries');
+
+const projectAdd = async (req, res, next) => {
+  try {
+    const {
+      name,
+      description,
+      imgUrl,
+      githubLink,
+      websiteLink,
+      projectType,
+      cohortId,
+    } = req.body;
```

Comment:
Why don't you just send `req.body` instead of defining all these variables and took 17 lines ?
you already have a validation middleware which will check the req.body keys and make sure the `req.body` keys are written right.

Reply:
thank you for this important note, I modify it